NuTrello/NuTrello/Pages/Index.cshtml

```diff
@@ -3,8 +3,13 @@
 @{
     ViewData["Title"] = "Home page";
 }
-
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
+
+    @for (int i = 0; i < 10; i++)
+    {
+        <h1>Bella är Bäst! Ingen Protäst!!</h1>
```

Comment:
hmmmmmmmmmmmmmmmmmmmmmmmmmmm.

Reply:
Darnit, jag trodde verkligen jag skulle få in den eftersom det är tidigt på morgonen :P 
Får vara mer sneaky nästa gång ;)